src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -195,6 +195,17 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
                 .AppendLine($"  fi")
                 .AppendLine("fi")
 
+                // Required for python 3.10 and 3.11
+                .AppendLine($"PIP_GCC_FLAGS=\"\"")
+                .AppendLine($"if [ \"$platformName\" = \"python\" ]; then")
+                .AppendLine($"  if [[ '{version}' == 3.10* ]] || [[ '{version}' == 3.11* ]]; then")
```

Comment:
Just to be sure I get it: do we expect the issue to not happen in the next Python version? Or do we just not know?
If we expect it should happen in 3.12, then having a logic that targets "greater or equal" versions would reduce the risk that we just forget about this case when 3.12 arrives. That said, if we reasonably believe that this might not be an issue in 3.12, please feel free to ignore.

Reply:
We'd have to support this in Oryx to be able to test. Since the library that would be responsible for addressing header files is only supported for 3.10 and 3.11, I figured that would be the versions we have this work around for:

- https://packages.debian.org/sid/python3.10-dev
- https://packages.debian.org/sid/python3.11-dev

Reply:
OK seems fair, thanks.